aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Configuring authentication for minimal APIs
+author: captainsafia
+description: Learn how to configure authentication and authorization in minimal API apps
+ms.author: safia
+monikerRange: '>= aspnetcore-7.0'
+ms.date: 10/17/2022
+uid: fundamentals/minimal-apis/security
+---
+
+# Authentication and authorization in minimal APIs
+
+## Key concepts in authentication and authorization
+
+Minimal APIs support all the authentication and authorization options available in ASP.NET and provide some additional functionality to improve the experience for working with authentication. Authentication is the process of determining a user's identity. Authorization is the process of determining whether a user has access to a resource. Both authentication and authorization scenarios share similar implementation semantics in ASP.NET Core. Authentication is handled by the authentication service, [IAuthenticationService](/dotnet/api/microsoft.aspnetcore.authentication.iauthenticationservice), which is used by authentication [middleware](/aspnet/core/fundamentals/middleware). Authorization is handled by the authorization service, [IAuthorizationService](/dotnet/api/microsoft.aspnetcore.authorization.iauthorizationservice), which is used by the authorization middleware.
```

Comment:
nit: ASP.NET Core > ASP.NET

Reply:
`@DamianEdwards`  won't be happy dropping Core on ASP.NET Core.

Reply:
?? Greater than, not ->